ccl/changefeedccl: TestChangefeedSchemaChangeBackfillCheckpoint failed

ccl/changefeedccl.TestChangefeedSchemaChangeBackfillCheckpoint [failed](https://tanzanite.cluster.engflow.com/invocations/default/adb84329-fd75-46ae-9a59-ee208d9f9db6?testReportRun=22&testReportShard=10&testReportAttempt=1#targets-Ly9wa2cvY2NsL2NoYW5nZWZlZWRjY2w6Y2hhbmdlZmVlZGNjbF90ZXN0) on master @ [3afb6935d0ef3de7c0d44cfb3cd54f312752c186](https://github.com/cockroachdb/cockroach/commits/3afb6935d0ef3de7c0d44cfb3cd54f312752c186):


```
=== RUN   TestChangefeedSchemaChangeBackfillCheckpoint
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestChangefeedSchemaChangeBackfillCheckpoint2280321320
    test_log_scope.go:81: use -show-logs to present logs inline
    changefeed_test.go:2184: random seed: -8057968037395150641
    changefeed_test.go:2454: -- test log scope end --
test logs left over in: outputs.zip/logTestChangefeedSchemaChangeBackfillCheckpoint2280321320
--- FAIL: TestChangefeedSchemaChangeBackfillCheckpoint (52.26s)
=== RUN   TestChangefeedSchemaChangeBackfillCheckpoint/kafka
    helpers_test.go:868: making server as system tenant
    helpers_test.go:944: making kafka feed factory
    helpers_test.go:68: using legacy schema changer
    changefeed_test.go:2294: backfill timestamp: 1720775887.425882359,2147483647
    changefeed_test.go:2341: initial checkpoint: [/Table/106/1/{101/0-501/0}]
    changefeed_test.go:2400: condition failed to evaluate within 45s: from changefeed_test.go:2404: waiting for second checkpoint
    --- FAIL: TestChangefeedSchemaChangeBackfillCheckpoint/kafka (52.25s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=22</code>
 - <code>shard=10</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #119375 ccl/changefeedccl: TestChangefeedSchemaChangeBackfillCheckpoint failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-23.1]
</p>
</details>
/cc @cockroachdb/cdc
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedSchemaChangeBackfillCheckpoint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>


Jira issue: CRDB-40256

--- a/pkg/ccl/changefeedccl/changefeed_test.go
+++ b/pkg/ccl/changefeedccl/changefeed_test.go
@@ -2216,7 +2216,7 @@ func TestChangefeedSchemaChangeBackfillCheckpoint(t *testing.T) {
 		}
 
 		// Setup changefeed job details, avoid relying on initial scan functionality
-		baseFeed := feed(t, f, `CREATE CHANGEFEED FOR foo WITH resolved='100ms', no_initial_scan`)
+		baseFeed := feed(t, f, `CREATE CHANGEFEED FOR foo WITH resolved='100ms', min_checkpoint_frequency='100ms', no_initial_scan`)
 		jobFeed := baseFeed.(cdctest.EnterpriseTestFeed)
 		jobRegistry := s.Server.JobRegistry().(*jobs.Registry)
 
@@ -2247,6 +2247,7 @@ func TestChangefeedSchemaChangeBackfillCheckpoint(t *testing.T) {
 		testutils.SucceedsSoon(t, func() error {
 			prog := loadProgress()
 			if p := prog.GetHighWater(); p != nil && !p.IsEmpty() {
+				t.Logf("highwater: %s", p)
 				return nil
 			}
 			return errors.New("waiting for highwater")

